chainer/training/updaters/multiprocess_parallel_updater.py

```diff
@@ -124,6 +124,18 @@ def __init__(self, iterators, optimizer, converter=convert.concat_examples,
                 'requires NCCL.\n'
                 'Please reinstall chainer after you install NCCL.\n'
                 '(see https://github.com/chainer/chainer#installation).')
+        try:
+            cuda.cupy.cuda.driver.ctxGetCurrent()
+            _cuda_initialized = True
+        except Exception:
```

Comment:
How about catching `cuda.cupy.cuda.driver.CUDADriverError`?

Reply:
Thanks, I fixed this part!